src/Compilers/Core/Portable/Emit/AnonymousTypeKey.cs

```diff
@@ -21,7 +21,17 @@ internal struct AnonymousTypeKeyField : IEquatable<AnonymousTypeKeyField>
         /// </summary>
         internal readonly bool IsKey;
 
-        internal AnonymousTypeKeyField(string name, bool isKey = false)
+        internal static AnonymousTypeKeyField CreateField(string name)
+        {
+            return new AnonymousTypeKeyField(name, isKey: false);
+        }
+
+        internal static AnonymousTypeKeyField CreateField(string name, bool isKey)
```

Comment:
So, the main idea is that this version of CreateField is only called by VB, so we call "ToLower". Am I right? Then, it looks good to me. Thanks!